crates/app/src/app.rs

```diff
@@ -274,7 +274,7 @@ pub struct Cli {
     )]
     pub cache: CacheMode,
 
-    #[arg(long, global = true, help = "Force colored output")]
+    #[arg(long, global = true, env = "MOON_COLOR", help = "Force colored output")]
```

Comment:
@chadrik This is actually wrong, since `MOON_COLOR` accepts more than just bools.

Reply:
Ok, I removed this.  Let me know if there's anything else you'd like me to do here.  I could update the help string to point users at the env var:  `Force colored output [env: MOON_COLOR=]`.